components/alert/alert.css

```diff
@@ -1,38 +1,38 @@
 :root {
-    --error-red: #ef233c;
-    --warning-yellow: #ff9100;
-    --info-blue: #0077b6;
-    --success-green: #2b9348;
+  --error-red: #ef233c;
+  --warning-yellow: #ff9100;
+  --info-blue: #0077b6;
+  --success-green: #2b9348;
 }
 
 .alert-container {
-    display: flex;
-    flex-flow: column wrap;
-    gap: 1rem;
+  display: flex;
+  flex-flow: column wrap;
+  gap: 1rem;
 }
 
 .alert {
-    display: flex;
-    align-items: center;
-    width: 80%;
-    padding: 0.8rem;
-    gap: 0.4rem;
-    color: whitesmoke;
-    border-radius: 5px;
+  display: flex;
+  align-items: center;
+  width: 80%;
+  padding: 0.8rem;
+  gap: 0.4rem;
+  color: whitesmoke;
+  border-radius: 5px;
 }
 
 .alert-error {
-    background-color: var(--error-red);
+  background-color: var(>Text utiltieserror-red);
```

Comment:
Here, the error alert is barely visible. You can try changing the colors of the error alert component.
![image](https://user-images.githubusercontent.com/70570591/160931499-4bce3869-de5d-4753-95e3-21a4347209b3.png)